packages/react/src/themes/teams/components/Avatar/avatarStyles.ts

```diff
@@ -26,6 +27,7 @@ const avatarStyles: ComponentSlotStylesInput<AvatarProps, any> = {
     }
   },
   image: ({ variables: v }): ICSSInJSStyle => ({
+    boxSizing: 'border-box',
```

Comment:
I saw that you added the same thing in the image, that would mean that we don't need i there.

Reply:
yes, but this is, actually, necessary, as not only the `Image` component could be rendered here. Essentially, we would like to ensure that all sizing props defined for this slot will be applied in the expected way